.github/workflows/gh-lint.yaml

```
@@ -0,0 +1,152 @@
+---
+    # actionlint と ghalint を使って GitHub Actions の静的解析する
+    #
+    # - actionlint: https://github.com/rhysd/actionlint
+    # - ghalint: https://github.com/suzuki-shunsuke/ghalint
+    name: GitHub Actions Lint
```

yamllint

[warning] wrong indentation: expected 0 but found 4

(indentation)